internal/testrunner/test_config.go

```diff
@@ -14,7 +15,24 @@ type SkipConfig struct {
 	Reason string `config:"reason"`
 
 	// Link is a URL where more details about the skipped test can be found.
-	Link string `config:"link"`
+	Link packedURL `config:"link"`
+}
+
+type packedURL struct {
+	*url.URL
+}
+
+func (u *packedURL) Unpack(s string) error {
+	var err error
+	u.URL, err = url.Parse(s)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (u packedURL) String() string {
+	return u.URL.String()
 }
```

Comment:
Is this needed? As the `*url.URL` is embedded, `packedURL` should already have this method.

Reply:
That's true, packedURL should have already defined `String()` method because of `url.URL`.

I'll test it just in case, and update the PR.
Thanks!